conf/messages.en

```diff
@@ -821,6 +821,26 @@ otherOfficialsNino.error.format = Enter a National Insurance number in the corre
 
 otherOfficialsNino.checkYourAnswersLabel = National Insurance number
 
+# Other  Official Address Messages
+# ----------------------------------------------------------
+otherOfficialAddress.lookupPage.title = What is the other official''s home address? - Officials and nominees
+otherOfficialAddress.lookupPage.heading = What is {0}''s home address?
+otherOfficialAddress.lookupPage.manualAddressLinkText = Other official''s home address does not have a UK postcode
+
+otherOfficialAddress.selectPage.title = What is the other official''s full address? - Officials and nominees
```

Comment:
this message should be "**What is the other official''s full home address? - Officials and nominees**"